### PR TITLE
feature - view register screen

### PR DESCRIPTION
**Pull request by at-cuongdoan**

### CODE REVIEW

File: MyApp/View/Register/RegisterViewController.swift [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            pickerGender.frame = view.bounds
+            pickerGender.delegate = self
+            view.addSubview(pickerGender)
+        }
+    }
+}
+
+extension RegisterViewController: UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String: Any]) {
+        let image = info[UIImagePickerControllerOriginalImage] as! UIImage
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force casts should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+    func creatAlert(_ text: String) {
+        let alert = UIAlertController(title: kWarning, message: "\(text)", preferredStyle: .alert)
+        alert.view.tintColor = .black
+        alert.addAction(UIAlertAction(title: kOK, style: .default, handler: nil))
+        present(alert, animated: true, completion: nil)
+    }
+
+    @IBAction func pressRegisterButton(_ sender: UIButton?) {
+        let username = usernameTextField.text!
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        let password = passwordTextField.text!
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        let email = emailTextField.text!
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+        let doB = birthTextField.text!
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+        let gender = genderTextField.text!
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+        if let inputEmail = viewModel.isValidEmail(emailTextField.text!) {
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+            creatAlert(inputEmail)
+            return
+        }
+        if let inputUsername = viewModel.isValidUsername(usernameTextField.text!) {
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+            creatAlert(inputUsername)
+            return
+        }
+        if let inputPassword = viewModel.isValidPassword(passwordTextField.text!, confirmTextField.text!) {
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+            creatAlert(inputPassword)
+            passwordTextField.text = kEmpty
+            confirmTextField.text = kEmpty
+            return
+        }
+        if let inputAll = viewModel.isValidBirthGender(birthTextField.text!, genderTextField.text!) {
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: ERROR - Force unwrapping should be avoided.

File: MyApp/View/Register/RegisterViewController.swift [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+            return false
+        }
+        return true
+    }
+
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        var sv = scrollView.contentSize.height
+        sv = view.frame.height + 200
+        scrollView.contentSize.height = sv
+
```
Comment (at-daiho) [inline MyApp/View/Register/RegisterViewController.swift]: WARNING - There should be a space after //

**Comment by at-daiho** [comment]
Total linter violations in pull request: 13
Serious: 0
Warnings: 1

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-hieutran** [comment]
@at-cuongdoan fix conflict e ơi.

**Pull request closed without merging**